### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by OybekKayumov**
In this branch:
- Created new `.js`  file with 3 functions `add`, `update`, and `remove`.
- Call the `add` function on `enter` press or `click`.
- Call the `update` function on the `enter` press.
- Call the` remove` function on `trash icon` click.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -45,6 +30,55 @@ const renderTasks = () => {
...
+  document.querySelectorAll('.delete').forEach((delBtn) => {
+    delBtn.addEventListener('click', () => {
+      const id = Number(delBtn.parentNode.parentNode.id.split('-')[1]);
+
+      removeTask(tasks, id);
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+      delBtn.parentNode.parentNode.remove();
+    });
+  });
 };
```
Comment (Emmyn5600) [inline src/index.js:74]: - [ ]  Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

As you can use on the below image, the indexes are not being updated when the delete function is pressed. so you should update the remaining indexed after deleting a task in the array.

- [ ] When you select the task as completed it doesn't update the local storage, so by default the completed is` false` but you need to update to `true` when the task is done.

image

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @OybekKayumov 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne:

- All linters are passing :heavy_check_mark:
- Good commit history :heavy_check_mark:
- Good PR description :heavy_check_mark:

## Required Changes :recycle:

- [ ]  Check the comment under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (OybekKayumov) [reply src/index.js:74]: Thank you for feedback.
Added update all remaining items indexes.
Select the task as completed update the local storage to true will be add at last project.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Oybek, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**